.github/RELEASE_NOTES.template.md

```diff
@@ -1,12 +1,12 @@
-# Frequenz Common API Release Notes
+# Frequenz common API Release Notes
```

Comment:
Being pedantic, but I'd suggest `Common` with `C` in upper case. It would make it more consistent with the rest of the words in the heading.
Also, I think it's better if we treat `Frequenz Common API` as a proper name, and capitalize the first letter of every word in it, in general.

Reply:
Agreed, but this would have to be ideally fixed in repo-config to avoid having to correct it each time we update the templates.

Reply:
Makes sense. Do we have an issue for this in the repo-config repo?

Reply:
* https://github.com/frequenz-floss/frequenz-repo-config-python/pull/89

Will fix this PR too.

Reply:
Thanks!

Reply:
Thanks!